src/master/grafana/dashboards/InfiniBand/network-counters.json

```diff
@@ -746,5 +870,6 @@
   "timezone": "browser",
   "title": "Network Counters",
   "uid": "baxV7f9Gz",
-  "version": 3
+  "version": 1,
+  "weekStart": ""
 }
```

Comment:
new line?

Reply:
I can add. This was auto generated by grafana when I exported.

Reply:
addressed